tests/src/io.jsonwebtoken/jjwt-impl/0.11.5/build.gradle

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+
+import org.graalvm.internal.tck.TestUtils
+
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = TestUtils.testedLibraryVersion
+
+dependencies {
+    testImplementation "io.jsonwebtoken:jjwt-api:$libraryVersion"
+    testImplementation "io.jsonwebtoken:jjwt-impl:$libraryVersion"
+    testImplementation "io.jsonwebtoken:jjwt-jackson:$libraryVersion"
```

Comment:
- Tests that add GraalVM reachability metadata for `io.jsonwebtoken:jjwt-impl` must add `io.jsonwebtoken:jjwt-api` and `io.jsonwebtoken:jjwt-jackson`. 
- The other two dependencies are not convenient for unit testing. Should I completely separate them into three directories for separate unit testing?

Reply:
- Solved. Since I can't imagine using `io.jsonwebtoken:jjwt-api:0.11.5` alone, I don't see any need to separate its GraalVM reachability metadata.